kafka-bridge/src/main/docker/holdings-items-kafka-bridge.json

```diff
@@ -0,0 +1,15 @@
+{
+    "appName": "${APPLICATION_NAME}",
```

Comment:
Jeg ser meget gerne at det er det rigtige "application name" der står her, istedet for variablen, for nåe man sidder med et antal .json åbne er det nemmere at overskue hvilken app den hører til.

Det hjælper selvfølgeligt at json'en hedder noget passende istedet for app.json ;-)